mws/resource_mws_budgets.go

```diff
@@ -0,0 +1,61 @@
+package mws
+
+import (
+	"context"
+	"github.com/databricks/databricks-sdk-go/service/billing"
+	"github.com/databricks/terraform-provider-databricks/common"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func ResourceMwsBudget() common.Resource {
+	s := common.StructToSchema(billing.BudgetConfiguration{}, func(m map[string]*schema.Schema) map[string]*schema.Schema {
+		common.CustomizeSchemaPath(m, "display_name").SetValidateFunc(validation.StringLenBetween(1, 128))
+		for _, p := range []string{"account_id", "budget_configuration_id", "create_time", "update_time"} {
+			common.CustomizeSchemaPath(m, p).SetComputed()
+		}
+		return m
+	})
+	p := common.NewPairSeparatedID("account_id", "budget_configuration_id", "/")
```

Comment:
What is the reason for using `/` as separator? We can just use `NewPairID` instead that will use `|` as separator.